rts/pom.xml

```diff
@@ -31,7 +31,7 @@ Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
   <url>http://www.jboss.org/jbosstm</url>
 
   <properties>
-    <version.quarkus>1.8.3.Final</version.quarkus>
+    <version.quarkus>1.13.7.Final</version.quarkus>
```

Comment:
This should be removed and depend upon the Quarkus upgrade PR from over here: https://github.com/jbosstm/narayana/pull/1967

Reply:
If you like, you could update it here to 2.9.0.Final too as that is the version we are targeting for the next release? But to mention that assuming https://github.com/jbosstm/narayana/pull/1972 is merged, I think there will then be a merge conflict here. Perhaps you could instead try using https://github.com/jbosstm/narayana/pull/1972/commits/6d838530a0226cfd706309bb94ec6b2c6e47dc1b in your branch? I think there could be some TCK hang with 2.9.0.Final though.

Reply:
Apparent hanging is mentioned in https://github.com/jbosstm/narayana/pull/1967